main.go

```diff
@@ -69,13 +72,29 @@ func main() {
 	}
 	defer db.Close()
 
-	// var queueBackend queue.QueueBackend
-	// var storageBackend filestore.StorageBackend
+	var queueBackend queue.QueueBackend
+	switch config.Transport.Queue {
+	case "memory":
+		queueBackend = memoryqueue.NewQueue()
+	case "sqs":
+		// TODO: replace with SQS queue once it's implemented
+		queueBackend = memoryqueue.NewQueue()
+	default:
+		log.Fatal().Msgf("Invalid transport.queue: %s: Expected 'memory' or 'sqs'", config.Transport.Queue)
```

Comment:
@poundifdef Instead of a dummy transports that returns an error, I opted to just fail there and then since as soon as the transport starts it's going to fail annyway.

Let me know if you'd like me to implement the dummy transport instead.

Reply:
Makes sense